opennms-base-assembly/src/main/filtered/etc/report-templates/subreports/ResponseTimeSummary_Availability_Offenders_subreport.jrxml

```diff
@@ -13,13 +13,28 @@
 		<parameterDescription><![CDATA[The directory where all reports can be found]]></parameterDescription>
 		<defaultValueExpression><![CDATA["${install.dir}/etc/report-templates"]]></defaultValueExpression>
 	</parameter>
-	<parameter name="START_TIME" class="java.util.Date"/>
-	<parameter name="TIME_RANGE" class="java.lang.String">
-		<defaultValueExpression><![CDATA["7 Days"]]></defaultValueExpression>
+	<parameter name="DS_SURVEILLANCE_CATEGORY" class="java.lang.String">
+		<parameterDescription><![CDATA[Surveillance category to include in this report]]></parameterDescription>
+	</parameter>
+	<parameter name="TIME_RANGE_NUMBER" class="java.lang.Integer">
+		<parameterDescription><![CDATA[Report Time Range in Days]]></parameterDescription>
+		<defaultValueExpression><![CDATA[7]]></defaultValueExpression>
+	</parameter>
+	<parameter name="START_TIME" class="java.util.Date">
+		<defaultValueExpression><![CDATA[new java.util.Date(new java.util.Date().getTime() - (86400000L * $P{TIME_RANGE_NUMBER}))]]></defaultValueExpression>
```

Comment:
Yes. 
Each paramater not marked as `isForPrompting=false` is showing up. 
If a description is defined, a tooltip is shown to the user in the web ui.